docs/Previous-Releases.md

```diff
@@ -1,6 +1,10 @@
 **We strongly recommend that you use the [current version](https://github.com/transmission/transmission/releases/latest) of Transmission unless you have a legacy operating system version shown below.**
 
 If you have a legacy operating system version, this table shows the best Transmission version to use:
+ 
+ * **Mac OS X 10.10 (Yosemite), 10.11 (El Capitan)**
```

Comment:
nit: those versions are known as "OS X" without the "Mac"

Reply:
I think the author was just trying to match the existing entries.

Reply:
That is true.

@Coeur: Feel free to make a further update to your standard of correctness.

Reply:
@wally-crunk I'm not a member of Transmission, I do not have write access to others' PRs. Best that you do perform the edits. :)

Reply:
OK, thank you for reporting the document was missing information.
I've made the improvements in #6601